lib/shelley/test/data/cardano-node-shelley/shelley-genesis.yaml

```diff
@@ -47,12 +47,12 @@ maxLovelaceSupply: 1000000000000000000
 protocolMagicId: 764824073
 networkMagic: 764824073
 networkId: Mainnet
-epochLength: 200
+epochLength: 50
 staking:
 slotsPerKESPeriod: 86400
 slotLength: 0.2
 maxKESEvolutions: 90
-securityParam: 10
```

Comment:
Hm, it seemed we used k=10 for shelley and k=8 for byron... Weird, I wouldn't have thought that would work...